layers/+lang/c-c++/README.org

```diff
@@ -112,6 +115,8 @@ doesn't complain about missing header files.
 | ~SPC m g A~ | open matching file in another window (e.g. switch between .cpp and .h) |
 | ~SPC m D~   | disaster: disassemble c/c++ code                                       |
 | ~SPC m r~   | srefactor: refactor thing at point.                                    |
+| ~SPC m p~   | Project / Build system management                                      |
```

Comment:
Yes, you're not actually documenting any of the keybindings. This is rather lazy.